engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableMaterial.java

```diff
@@ -25,7 +25,7 @@
 /**
  * TODO: Add javadocs
  */
-public class EnableMaterial implements StateChange {
+public final class EnableMaterial implements StateChange {
```

Comment:
As far as I know, it's for denoting that this state change implementation can not used as a subclass in another state change implementation. There were already `final` keywords in other state changes. So for sake of consistency, I added that.